### PR TITLE
Update README with example and docs

### PR DESCRIPTION
**Pull request by johnbrett**
- Added Example
- Add docs

### CODE REVIEW

**Comment by johnbrett** [comment]
I would like to take maintain this module @hueniverse

File: README.md [hunk 1]
```diff
@@ -1,7 +1,46 @@
...
 Lead Maintainer - [Eran Hammer](https://github.com/hueniverse)
+
+
+## Example
+
+```javascript
+var Http = require('http');						
+var Subtext = require('subtext');
+
+Http.createServer(function(request, response) {
```
Comment (gergoerdosi) [inline README.md]: Add a space after `function` and use 4 spaces for indentation: https://github.com/hapijs/contrib/blob/master/Style.md

File: README.md [hunk 2]
```diff
@@ -1,7 +1,46 @@
...
+
+```javascript
+var Http = require('http');						
+var Subtext = require('subtext');
+
+Http.createServer(function(request, response) {	
+
+  	Subtext.parse(request, null, { parse: true, output: 'data' }, function(err, parsed) {
+
+  		response.writeHead(200, {'Content-Type': 'text/plain'});
```
Comment (gergoerdosi) [inline README.md]: Space after `{` and before `}`.

**Comment by johnbrett** [comment]
Thanks for advice @mtharrison and @gergoerdosi, all addressed.

**Comment by hueniverse** [comment]
You are all set with the required permissions.

**Comment by johnbrett** [comment]
Thanks @hueniverse! :)

**Pull request merged**